src/ray/core_worker/common.h

```diff
@@ -68,6 +68,18 @@ class TaskArg {
   const std::shared_ptr<Buffer> data_;
 };
 
+/// Information of a task
+struct TaskInfo {
+  /// The ID of task.
+  const TaskID task_id;
+  /// The driver ID.
+  const DriverID driver_id;
+  /// Whether this task is to create an actor.
+  const bool is_actor_creation_task;
+  /// Whether this task is to execute a function of an actor.
+  const bool is_actor_task;
```

Comment:
Use an enum `Type` to indicate the type of this task?

Reply:
Great idea!